index.bs

```diff
@@ -286,6 +286,7 @@ spec:css-syntax-3;
     interface Credential {
       readonly attribute USVString id;
       readonly attribute DOMString type;
+      static boolean isConditionalMediationAvailable();
```

Comment:
In reviewing the WebIDL spec, I am wondering whether declaring `isConditionalMediationAvailable()` as `static` is what we want.  WebIDL says this:
> ... static operations are ones that are not associated with a particular instance of the interface on which it is declared, and is instead associated with the interface itself.

Will it not be possible for there to be multiple PublicKeyCredential, PasswordCredential, FederatedCredential objects floating around at concurrently (all based on Credential), and should not each type ought to implement (or not) its own  `isConditionalMediationAvailable()` method ?

IIUC, using `static` would mean there'd be only the one implementation on the `Credential` interface itself.

Reply:
This definitely has to be `static` or you'd need an instance of an object implementing the interface. I can see an argument for not including it in the base class, but then we'd have to a) say it must always be implemented for all subclasses and b) add it on `PasswordCredential` and `FederatedCredential`.

Reply:
@nsatragno has gently pointed out to me that I am indeed misunderstaing things here and that the way it is currently spec'd is correct. doh!